pkg/api/vlabs/validate_test.go

```diff
@@ -4139,7 +4139,7 @@ func TestValidateProperties_OrchestratorSpecificProperties(t *testing.T) {
 		}
 	})
 
-	t.Run("Should not support os type other than linux for single stack ipv6 and dual stack feature", func(t *testing.T) {
+	t.Run("Should not support os type other than linux along with kubernetes version less than 1.19 for single stack ipv6 and dual stack feature", func(t *testing.T) {
```

Comment:
The wording here was alittle akward.  Maybe something like:

```suggestion
	t.Run("Should not support os type other than linux for versions less than 1.19 for single stack ipv6 and dual stack feature", func(t *testing.T) {
```